ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/influxdb.yml

```diff
@@ -0,0 +1,31 @@
+influxdb:
+  destinations:
+    - name: test
+      vrf: test
+      database: test
+      policy: test
+      url: https://influx_test.localhost
+      username: test
+      password: test
+      encryption: 0
+    - name: test1
+      vrf: test1
+      database: test1
+      policy: test1
+      url: https://influx_test1.localhost
+      username: test1
+      password: test1
+      encryption: 7
```

Comment:
This can not be configure without encrypted value.

Reply:
Added encrypted password.